src/components/Messages/Message/index.js

```diff
@@ -205,6 +214,7 @@ export const Message = memo(({
 				resetLastAction,
 				actionsVisible: message.actionsVisible ? message.actionsVisible : false,
 				showPostChatUrl: showPostChatFeedback({ chatClosed, message, postChatUrl }),
+				isChatClosed: isChatClosed(message),
```

Comment:
We can just use `isChatClosed: message.t === MESSAGE_TYPE_LIVECHAT_CLOSED` So no need to create separate function for it 😊